src/api/Feed.js

```diff
@@ -494,6 +494,50 @@ class Feed extends Base {
         });
     };
 
+    /**
+     * Updates a task in the new API
+     *
+     * @param {BoxItem} file - The file to which the task is assigned
+     * @param {string} task - The update task payload object
+     * @param {Function} successCallback - the function which will be called on success
+     * @param {Function} errorCallback - the function which will be called on error
+     * @return {void}
+     */
+    updateTaskNew = (
+        file: BoxItem,
+        task: TaskUpdatePayload,
+        successCallback: () => void = noop,
+        errorCallback: ErrorCallback = noop,
+    ) => {
+        if (!file.id) {
+            throw getBadItemError();
+        }
+
+        this.id = file.id;
+        this.errorCallback = errorCallback;
+        this.tasksNewAPI = new TasksNewAPI(this.options);
+        this.updateFeedItem({ isPending: true }, task.id);
+        this.tasksNewAPI.updateTask({
+            file,
+            task,
+            successCallback: (taskData: Task) => {
+                this.updateFeedItem(
+                    {
+                        ...taskData,
```

Comment:
does this also need to merge with the old task data (for nested values like assigned_to)?

Reply:
@alexkrolick Yep, however it is outside the scope of this task. Once assignment stuff is figured out appropriate logic can be added here.